demo/data.js

```diff
@@ -510,8 +518,14 @@ export const demoData = {
         },
         {
           id: "9",
+<<<<<<< HEAD
```

Comment:
A little rebase is needed :)
I made a PR to update demo data in v25 (https://github.com/odoo/o-spreadsheet/pull/5563/files) so you can remove the next commit. Then you can add a commit with an update to v26 (it will make the diff lower).
Thanks !